### DISCUSSION TITLE
multiple filter string

### DISCUSSION THREAD

**Discussion by molitona** [projectdiscovery/httpx#1107] [Q&A] [answered]
Hi

-fs xx -fs yy -fs zz don't work, why?

**Comment by olearycrew** [comment, marked as answer]
@molitona this works for me. Example:

`httpx -u http://boleary.dev -ms 'brendan' -ms 'olearycrew'`

<img width="963" alt="CleanShot 2023-04-19 at 12 22 14@2x" src="https://user-images.githubusercontent.com/6044920/233138593-5cb7ad54-d0b9-4bd0-929f-569d2da48d20.png">


`httpx -u http://boleary.dev -ms 'brendan' -ms 'olearycrew' -ms 'forsurenotonmywebsite'`

<img width="1292" alt="CleanShot 2023-04-19 at 12 21 48@2x" src="https://user-images.githubusercontent.com/6044920/233138502-96eefde7-b1e9-4a8f-b08e-2ef212999616.png">

**Comment by molitona** [comment]
Hi @olearycrew,

sorry for the late response.

it didn't work for me many times. i'll try to investigate why.

**Comment by molitona** [comment]
@olearycrew i tried it now and failed. in fact i put "-fs" in the post and you mistakenly used "-ms". I want to remove xx & yy & ZZ not match them
